plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/TestingPinotCluster.java

```diff
@@ -204,6 +214,13 @@ public void addRealTimeTable(InputStream realTimeSpec, String tableName)
         throw exception;
     }
 
+    private static String controllerAuthToken()
```

Comment:
Please reuse the method in `PinotBasicAuthenticationProvider`.

Reply:
I would have to relax the access in the method definition and make it static just because of this test, I wouldn't do that, Now that I use OKHTTP I can delegate to that instead. Let me know if you agree

Reply:
You can use `@VisibleForTesting` annotation for such case.